Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Buttons/Controls Toolkit/Dialog Buttons/KryptonAbortDialogButton.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Krypton.Toolkit.Suite.Extended.Language.Model;
```

Comment:
Can you change your sorting order so that System usings are first, then a gap , then others are included. 
- There is a reason to this madness: I have found in the past, that the C# Compiler caches the '`System.* usings` first, so when compiling large projects, just this simple thing can improve the throughput of the processing.

Reply:
I'll turn off automatic sorting of usings